src/test.test.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const todoArr = [];
+const input = document.createElement('input');
+input.value = 'one';
+
+const dataObj = {
+  description: input.value,
+  completed: false,
+};
+
+const Add = () => {
+  if (input.value.length !== 0) {
+    dataObj.description = input.value;
+
+    for (let i = 0; i < todoArr.length; i += 1) {
+      todoArr[i].index = i + 1;
+    }
+    todoArr.push(dataObj);
+
+    localStorage.setItem('data', JSON.stringify(todoArr));
+  }
+};
```

Comment:
- [OPTIONAL] It is good practice to keep your tests and functions in separate files. You can just import the functions into the test.js file and test the there. What do you think 🤔 ?